packages/ui/react-ui-editor/src/components/Markdown/markdownTheme.ts

```diff
@@ -164,6 +165,10 @@ export const markdownTheme = {
   '& .cm-yLineSelection': {
     margin: '0',
   },
+  '.cm-link': {
+    color: cyan,
+    textDecoration: 'underline',
```

Comment:
We should aim to align this with our Link component. I think ideally we should define a `.link` class name in theme.css.

In the short term, we could apply the literal properties here for now.
```suggestion
    color: 'rgb(20 89 208)',
    textDecorationLine: 'underline',
    textDecorationThickness: '1px',
    textUnderlineOffset: '2px',
    borderRadius: '.125rem'
```